app/views/stage/index.html.erb

```diff
@@ -80,56 +90,168 @@
+					<tr slot="header">
+						<th></th>
+						<th data-orderable="false">Name</th>
+						<th data-orderable="false">
+							Dist.
+							<span class="help-tooltip" data-toggle="static-popover" data-placement="bottom" data-content="Bacon ipsum dolor amet spare ribs cow prosciutto beef ribs venison. Sirloin bacon kielbasa chuck hamburger ball tip venison strip steak bresaola pork chop kevin."><i class="glyphicon glyphicon-question-sign"></i></span>
+						</th>
+						<th data-orderable="false">
+							V. Gain
+							<span class="help-tooltip" data-toggle="static-popover" data-placement="bottom" data-content="Bacon ipsum dolor amet spare ribs cow prosciutto beef ribs venison. Sirloin bacon kielbasa chuck hamburger ball tip venison strip steak bresaola pork chop kevin."><i class="glyphicon glyphicon-question-sign"></i></span>
+						</th>
+						<th data-orderable="false">
+							V. Loss
+							<span class="help-tooltip" data-toggle="static-popover" data-placement="bottom" data-content="Bacon ipsum dolor amet spare ribs cow prosciutto beef ribs venison. Sirloin bacon kielbasa chuck hamburger ball tip venison strip steak bresaola pork chop kevin."><i class="glyphicon glyphicon-question-sign"></i></span>
+						</th>
+						<th colspan="2" data-orderable="false">
+							Location Info
+							<span class="help-tooltip" data-toggle="static-popover" data-placement="bottom" data-content="Bacon ipsum dolor amet spare ribs cow prosciutto beef ribs venison. Sirloin bacon kielbasa chuck hamburger ball tip venison strip steak bresaola pork chop kevin."><i class="glyphicon glyphicon-question-sign"></i></span>
+						</th>
+					</tr>
+					<template slot="row" scope="vm">
+						<td><i class="glyphicon glyphicon-flag"></i></td>
+						<td>{{ vm.row.name }}</td>
+						<td>{{ vm.row.distance }}</td>
+						<td>{{ vm.row.verticalGain }}</td>
+						<td>{{ vm.row.verticalLoss }}</td>
+						<td><a data-toggle="modal" data-target="#js-split-modal" v-on:click="vm.$emit('edit')">Edit Split</a></td>
+						<td><a v-on:click="vm.$emit('remove')"><i class="glyphicon glyphicon-remove-circle"></i></a></td>
+					</template>
+				</data-tables>
+			</div>
+			<div class="col-xs-12">
+				<p class="pull-left">
+					<button type="button" class="btn btn-success" data-toggle="modal" data-target="#js-split-modal" v-on:click="modalData = blank()">Add New Split&nbsp;<i class="glyphicon glyphicon-plus"></i></button>
+					<button type="button" class="btn btn-default">Import GPX File</button>
+				</p>
+				<router-link to="/splits" class="btn btn-primary pull-right">Continue&nbsp;<i class="glyphicon glyphicon-arrow-right"></i></router-link>
+			</div>
 		</div>
 	</article>
-	<split-modal :isNew="modalNew" :split="modalSplit" v-on:new="eventData.splits.push( modalSplit )"></split-modal>
+	<edit-modal v-model="modalData" :validator="isValid" v-on:done="!( modalData._dtid ) && eventData.splits.push( modalData )" inline-template>
```

Comment:
Github is complaining about this && for some reason.  Probly thinks its HTML. C'mon Github...